machine.go

```diff
@@ -150,12 +150,8 @@ type Machine struct {
 }
 
 // Create a new machine object with the auto backend
-func NewMachine() *Machine {
-	m, err := NewMachineWithBackend("auto")
-	if err != nil {
-		panic(err)
-	}
-	return m
+func NewMachine() (*Machine, error) {
```

Comment:
this API isn't used by debos or fakemachine, so it is OK to modify
also, this should be used for the `CreateMachine` function in #76